src/handlers/authHandler.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  CognitoUserPool,
+  CognitoUser,
+  AuthenticationDetails,
+  CognitoUserAttribute,
+} from "amazon-cognito-identity-js";
+import dotenv from "dotenv";
+import userModel from "../models/userModel";
+import { getRandomPokemonId } from "../utils/randomStartingPokemon";
+
+dotenv.config();
+
+const userPool = new CognitoUserPool({
+  UserPoolId: process.env.COGNITO_USER_POOL_ID || "",
+  ClientId: process.env.COGNITO_CLIENT_ID || "",
+});
```

Comment:
I'd recommend creating an `envs.ts` file in your root to contain all of your environment variables. At the moment you only have two or three, but as the application grows you will have many more. 
Another critical thing to note here is that - without these envs you cannot handle logins into your system, this means that users would not be able to connect.. lets think of a mechanism that will notify you (or any other developer that is debugging this system) that these envs are missing, could be a life saver in a production environment.